cmd/harmony/flags.go

```diff
@@ -776,6 +782,10 @@ func applyConsensusFlags(cmd *cobra.Command, config *harmonyConfig) {
 	} else if cli.IsFlagChanged(cmd, legacyConsensusMinPeersFlag) {
 		config.Consensus.MinPeers = cli.GetIntFlagValue(cmd, legacyConsensusMinPeersFlag)
 	}
+
```

Comment:
It was my fault. In applyConsensusFlags, it would better to be 

```
if config.Consensus == nil && cli.HasFlagsChanged(cmd, consensusValidFlags) {
	cfg := getDefaultConsensusConfigCopy()
	config.Consensus = &cfg
}
```

Would you please update this change for me?

Reply:
done.